source/engine/graphics/api/vulkan/RenderGraph.cpp

```diff
@@ -284,10 +281,14 @@ void RenderGraph::Execute(const PerFrameRenderState &frameData, const Vector2& d
 
     SetViewportAndScissorFullWindow(cmd, dimensions);
 
-    for (auto& shadowMappingPass : renderGraph.shadowPasses)
+    auto instanceData = PerFrameInstanceData{};
+
+    for (auto [index, shadowMappingPass] : std::views::enumerate(renderGraph.shadowPasses))
     {
+        instanceData.isOmniDirectional = index < renderGraph.stateData.omniDirLightsCount ? true : false;
```

Comment:
Ternary is redundant here since the result of < is a bool.

Reply:
Thanks!